docs/user/integration/dynatrace/README.md

```diff
@@ -34,6 +34,7 @@ With the Kyma Telemetry module, you gain even more visibility by adding custom s
 - The [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) is [enabled](https://kyma-project.io/#/02-get-started/01-quick-install)
 - Active Dynatrace environment with permissions to create new access tokens
 - A Secret in the respective namespace in the Kyma cluster, holding the access tokens and endpoints for the Dynatrace instance. In this guide, the Secret is named `dynakube` and the namespace `dynatrace` as illustrated in this [example](https://github.com/kyma-project/telemetry-manager/blob/main/docs/user/integration/dynatrace/secret-example.yaml).
+  >**NOTE:** If you don't have the required access tokens you can create the Secret [later](#create-access-token-and-secret).
```

Comment:
```suggestion
  >**NOTE:** If you don't have the required access tokens, you can create the Secret [later](#create-access-token-and-secret).
```

Reply:
But really, if creating the Secret is part of this guide, why keep it as prerequisite at all?

Reply:
If the user already has access token he can create under pre-requisites. If he does not know how to create access tokens or does not know the scopes then he can create it later.

Reply:
I'm not sure I understand. Purpose of the guide is to instruct them how to create access tokens, isn't it? So the assumption is that they don't have them yet. Otherwise half of the guide is not required. But maybe I misunderstand the purpose and our assumptions for this guide.